setup-app.rb

```diff
@@ -4,12 +4,14 @@
 require 'fileutils'
 
 if ARGV.size != 1
-    puts "usage: setup-app.rb <config.yml>"
+    puts "usage: #{File.basename(__FILE__)} /path/to/nginx-mapping.yaml"
```

Comment:
I'm [renaming this script](https://github.com/akash1810/homebrew-devtools/blob/master/Formula/dev-nginx.rb#L10) in the brew formula to namespace it a little, use `File.basename(__FILE__)` to get an accurate name.